applications/nrf5340_audio/broadcast_sink/Kconfig.defaults

```diff
@@ -50,6 +50,11 @@ config BT_PAC_SNK
 config BT_AUDIO_RX
 	default y
 
+config BT_PER_ADV_SYNC_TRANSFER_RECEIVER
```

Comment:
Check this config to see if we need to enable the scan delegator feature or not.